src/caches/adams_bashforth_moulton_caches.jl

```diff
@@ -349,12 +349,17 @@ function alg_cache(alg::VCAB3,u,rate_prototype,uEltypeNoUnits,uBottomEltypeNoUni
   k2 = zeros(rate_prototype)
   k3 = zeros(rate_prototype)
   k4 = zeros(rate_prototype)
-  ϕstar_nm1 = Array{typeof(rate_prototype)}(1,3)
   grid_points = zeros(typeof(t),3)
   c = zeros(typeof(t),3,3)
   g = zeros(typeof(t),3)
   ϕ_n = Array{typeof(rate_prototype)}(1,3)
+  ϕstar_nm1 = Array{typeof(rate_prototype)}(1,3)
   ϕstar_n = Array{typeof(rate_prototype)}(1,3)
+  for i in 1:3
+    ϕ_n[i] = zeros(rate_prototype)
+    ϕstar_nm1[i] = zeros(rate_prototype)
+    ϕstar_n[i] = zeros(rate_prototype)
```

Comment:
doesn't look like these should alias. Try printing out the `ϕstar_n` values and see if every vector is the same.